src/DependencyInjection/LeezyPheanstalkExtension.php

```diff
@@ -22,10 +25,11 @@ class LeezyPheanstalkExtension extends Extension
     public function load(array $configs, ContainerBuilder $container)
     {
         $configuration = new Configuration();
-        $config = $this->processConfiguration($configuration, $configs);
+        $config        = $this->processConfiguration($configuration, $configs);
 
         $loader = new Loader\XmlFileLoader($container, new FileLocator(__DIR__.'/../Resources/config'));
         $loader->load('services.xml');
+        $loader->load('commands.xml');
 
         if (!$config['enabled']) {
```

Comment:
I just removed this option : https://github.com/armetiz/LeezyPheanstalkBundle/commit/86b5ecd203956b8ce64537942a089e74b001f64e

Regards